lib/ansible/modules/network/checkpoint/cp_mgmt_network.py

```diff
@@ -144,48 +141,41 @@
     type: bool
   ignore_errors:
     description:
-      - Apply changes ignoring errors. You won't be able to publish such a changes. If ignore-warnings flag was
-        omitted - warnings will also be ignored.
+      - Apply changes ignoring errors. You won't be able to publish such a changes. If ignore-warnings flag was omitted - warnings will also be ignored.
     type: bool
-  new_name:
-    description:
-      - New name of the object.
-    type: str
 extends_documentation_fragment: checkpoint_objects
 """
 
 EXAMPLES = """
 - name: add-network
-  cp_network:
+  cp_mgmt_network:
     name: New Network 3
     nat_settings:
       auto_rule: true
       hide_behind: ip-address
       install_on: All
       ip_address: 192.0.2.1
       method: static
-    state: present
```

Comment:
It's a best practice to keep `state: present` in the play example, please update it